docs/pages/cli-docs.mdx

```diff
@@ -532,22 +532,37 @@ registered devices using [`tsh mfa ls`](cli-docs.mdx#tsh-mfa-ls).
 
 ## tctl
 
-`tctl` is an admin CLI tool used to administer a Teleport cluster. It connects
-to an Auth Server, meaning that it must be run on the same host with the role
-`auth` . If you run `tctl` on a non- `auth` node it will show an error.
+`tctl` is an admin CLI tool used to administer a Teleport cluster. 
 
 `tctl` allows a cluster administrator to manage all resources in a cluster
 including nodes, users, tokens, and certificates.
 
 `tctl` can also be used to modify the dynamic configuration of the cluster, like
 creating new user roles or connecting trusted clusters.
 
+By default, `tctl` connects to a local Auth server, meaning it will fail if you 
+attempt to run `tctl` on a non-`auth` node.
+
+However `tctl` can also connect to a remote Auth server if the `--identity` and
+`--auth-server` flags are used.
+
+<Admonition type="note">
+  Note that when a `tctl` command is run locally on an Auth server, the audit logs will 
+  show that it was performed by the Auth server itself. To properly audit admin actions
+  at scale, it is important to limit direct ssh access to the Auth server with 
+  [Access Controls](./access-controls/introduction.mdx) and ensure that admins use `tctl` 
+  remotely with the `--identity` flag instead.
+</Admonition>
+
 ### tctl Global Flags
 
 | Name | Default Value(s) | Allowed Value(s) | Description |
 | - | - | - | - |
 | `-d, --debug` | none | none | Enable verbose logging to stderr |
 | `-c, --config` | `/etc/teleport.yaml` | **string** filepath | Path to a configuration file |
+| `--auth-server` | none | `host:port` | `tctl` attempts to connect to a specific auth/proxy address instead of local auth |
+| `-i, --identity` | none | **string** filepath | Path to an identity file, must be provided to make remote connections to auth |
```

Comment:
https://github.com/gravitational/teleport/blob/82185f232ae8974258397e121b3bc2ed0c3729ed/tool/tctl/common/tctl.go#L124 

Thanks for correcting that - I appreciate any improvements to our CLI section docs. Many are out of date or missing some functionalities.

Reply:
Follow-up: is there an equivalent to YARD, JavaDoc, etc. that we can use in CLI or other spots to autogenerate outputs?

Reply:
It would likely be dependent on the command line parser specific, but I don't see anything like that for `https://github.com/alecthomas/kingpin`. However we could improve the [help command](https://github.com/alecthomas/kingpin#custom-help) on each CLI with up to date and thorough descriptions.

IMO if we really step up these help commands, the cli reference will be redundant. We could instead follow the [api docs](https://goteleport.com/docs/ver/6.2/reference/api/architecture/) model and rely on the help command for most documentation, and then add `Getting Started` and `Architecture` sections for other CLI documentation that doesn't fit nicely in the help reference.

Reply:
Yeah, I like that approach quite a bit more. Less effort.

Reply:
There's actually an open issue about auto generating CLI docs from last year if you want to push the discussion forward.